minimisers/NS/nestedSampler.m

```diff
@@ -73,7 +73,16 @@
 % initialise array of samples for posterior
 nest_samples = zeros(1,D+1);
 coder.varsize('nest_samples',[1e5 50],[1 1]);
-    
+
+% check certain values are positive integers or zero
+if mod(nMCMC, 1) ~= 0 || nMCMC < 0
+    error('Error... nMCMC must be an integer >= 0')
```

Comment:
Message can be cleaned up, maybe

`error('NS Error: nMCMC must be an integer >= 0')`